### PR TITLE
Add force option to `setup-local-client.sh`

### PR DESCRIPTION
**Pull request by samveen**
Fix for #1783 

I've added `-f|--force` option to setup-local-client.sh for non-interactive usage. 

Caveat: I've used bash-2.04 compliant syntax. If this assumption about the shell is incorrect, please let me know.

Additional minor fixes:
- Indentation was set to 2 spaces: fixed non-compliant lines
- Fixed multiple spaces intra-line
- Redirected output of `cd -` as it leads to confusion in the output.

### CODE REVIEW

File: xCAT-server/share/xcat/scripts/setup-local-client.sh [hunk 1]
```diff
@@ -15,15 +15,31 @@ umask 0077 #nothing make by this script should be readable by group or others
 if [ -z "$XCATDIR" ]; then
   XCATDIR=/etc/xcat
 fi
+# if `-f`|`--force` option is supplied, set a variable and remove option from paraameters
+# This allows use of the old code without modification
+FORCE=0
+declare -a ARGS
```
Comment (immarvin) [inline xCAT-server/share/xcat/scripts/setup-local-client.sh]: this script might be run in "ksh" on aix, the "declare" is not supported
````
declare:  not found.
````

Comment (samveen) [inline xCAT-server/share/xcat/scripts/setup-local-client.sh]: @immarvin Can you confirm ksh version? I'll alter the script to match shell syntax intersection.

Comment (immarvin) [inline xCAT-server/share/xcat/scripts/setup-local-client.sh]: hi @samveen ,
````
# oslevel -r
7100-03
# echo $SHELL
/usr/bin/ksh
# lslpp -l | grep ksh
  bos.rte.shell             7.1.3.15  COMMITTED  Shells (bsh, ksh, csh)
  bos.rte.shell             7.1.3.15  COMMITTED  Shells (bsh, ksh, csh)
````

**Comment by samveen** [comment]
@immarvin I've just removed the `declare -a` which wasn't strictly necessary. Please check again.

**Comment by immarvin** [comment]
agree to merge

**Pull request merged**